### PR TITLE
PE-735: Stked ETH support in auction demo keeper

### PR DESCRIPTION
**Pull request by talbaneth**

### CODE REVIEW

**Comment by godsflaw** [comment]
```haskell
          WSTETH-A auction 1

            Auction Tab:        384475.541109249036437299 Dai
            Auction Lot:        157.935961636117181137
            Configured Lot:     between 0.0 and 253.668154394878651014
            Debt to Cover:      622608.549397423994502453 Dai
            Slice to Take:      157.935961636117181137
            Auction Price:      3942.15822 Dai

            Cost of lot:        607422.975021846251034269 Dai
            Minimum profit:     15565.213734935599862561 Dai

            Uniswap proceeds:   657435.430659250692977799 Dai
            Less min profit:    641870.2169243151


Attempting to take 157.935961636117181137 WSTETH-A from auction 1 at max price 3942.15822 Dai
previousNonce:  null

    Sending a transaction
    from 0x<REDACTED>
    to 0x49A33A28C4C7D9576ab28898F4C9ac7e52EA457A
    nonce 949
    gasLimit 1105974
    gasPrice 31.000000007 Gwei

Auction 1 Take Tx Hash 0x4f1efded7e42ad80c26a59b8f8237a4cabd733dadc0ae2d0c5a9164a7195f7ff
Deleting Auction ID: 1 with remaining tab 0.0 and lot 58.42652288174977126
```

**Review by godsflaw — APPROVED** [review]
LGTM 💪

File: src/keeper.js [hunk 1]
```diff
@@ -219,6 +250,7 @@ export default class keeper {
     this._uniswapCalleeAdr = collateral.uniswapCallee;
     this._uniswapLPCalleeAdr = collateral.uniswapLPCallee;
     this._oasisCalleeAdr = collateral.oasisCallee;
```
[the remark this answers is not in the document]
Comment (iamchrissmith) [reply src/keeper.js:252]: would also require changing on lines 35-37 

https://github.com/makerdao/auction-demo-keeper/pull/54/files?authenticity_token=aBGBhF%2B1IsxYfi2ilDHjQyTOBynkGSHwFS4hufIy2D7JDQanUE8vJQV96usaazoHR7G8oO10LQVdQpnUbVEZpw%3D%3D&file-filters%5B%5D=.js#diff-ca2e7daec1ddcaf20f4ce47d0083bb1bfce68c3733a34032969441b8832fb6c9R35-R37

We should also probably add `_wstETHCurveUniv3CalleeAddr = null;` there too

File: src/keeper.js [hunk 2]
```diff
@@ -219,6 +250,7 @@ export default class keeper {
...
+    this._wstETHCurveUniv3CalleeAddr = collateral.wstETHCurveUniv3Callee;
```
[the remark this answers is not in the document]
Comment (iamchrissmith) [reply src/keeper.js:253]: doing a search for these in the rest of the codebase, do we even need them? seems like we're using the `collateral` value below and that is what sets up the clippers

File: src/keeper.js [hunk 3]
```diff
@@ -200,6 +210,27 @@ export default class keeper {
...
             console.log('Not enough liquidity on OasisDEX\n');
           }
+
+        } else if (wstETHCurveUniv3) {
+          liquidityAvailability = `
+            Uniswap proceeds:   ${wstETHCurveUniv3Proceeds.receiveAmount} Dai
+            Less min profit:    ${minWstETHCurveUniv3Proceeds}\n`;
+          console.log(auctionSummary + liquidityAvailability);
+          if (Number(ethers.utils.formatUnits(costOfLot)) <= minWstETHCurveUniv3Proceeds) {
+            // tx executes only if the return amount also covers the minProfit %
```
Comment (iamchrissmith) [inline src/keeper.js:220]: is `minProfit` a percentage?

**Review by iamchrissmith — COMMENTED** [review]
couple of questions/comments (unimportant nits)

File: src/keeper.js (hunk 3 above)
Comment (talbaneth) [reply src/keeper.js:221]: Yeah not really. It is derived from minProfitPercentage from the configuration.

File: src/keeper.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (talbaneth) [reply src/keeper.js]: Done

File: src/keeper.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (talbaneth) [reply src/keeper.js:254]: Yeah, I see these aren't really usef. Possibly the keeper needs some cleanups but I am hesitant to do these now as we are working on a few things in parralel.

**Comment by talbaneth** [comment]
@iamchrissmith I addressed the stuff you mentioned, thanks a lot and did some minor changes. I don't really want to touch too much stuff at this point to avoid conflicts and so we can ship this asap for keepers to use (also we mostly do manual testing now).

**Review by iamchrissmith — APPROVED** [review]
lgtm

**Pull request merged**